front/src/modules/simulationResult/components/ChartHelpers/enableInteractivity.tsx

```diff
@@ -118,13 +123,16 @@ const updateChart = <
           .axisLeft<Date>(newY as d3.ScaleTime<number, number>)
           .tickFormat(d3.timeFormat('%H:%M:%S'))
       : d3.axisLeft(newY);
+  const axisRightY = d3.axisRight(newY2 as d3.ScaleTime<number, number>);
```

Comment:
Move this line below:
```suggestion
  if (chart.y2Axis) {
    const axisRightY = d3.axisRight(newY2 as d3.ScaleTime<number, number>);
    chart.y2Axis.call(axisRightY);
  }
```